sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -258,6 +258,11 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val PARQUET_RECORD_FILTER_ENABLED = buildConf("spark.sql.parquet.recordFilter")
+    .doc("Whether to allow the record-level filtering in Parquet for Spark to filter them.")
```

Comment:
Add something like: When `spark.sql.parquet.filterPushdown` is disabled, this config doesn't have any effect?

Reply:
Sure, let me add more details about it soon.